morango/certificates.py

```diff
@@ -34,9 +37,56 @@ class Certificate(mptt.models.MPTTModel, UUIDModelMixin):
     signature = models.TextField()
 
     # when we own a certificate, we'll have the private key for it (otherwise not)
-    private_key = PrivateKeyField(blank=True, null=True)
+    _private_key = PrivateKeyField(blank=True, null=True, db_column="private_key")
+
+    @property
+    def private_key(self):
+        return self._private_key
+
+    @private_key.setter
+    def private_key(self, value):
```

Comment:
I think it should happen at the later validation level -- throwing validation errors when a field's value is set probably isn't expected.